### PR TITLE
update shadow readme toadd explanation on shadow current capabilities

### PR DESCRIPTION
**Pull request by xps506**
### Motivation
fix a typo error and rephrase some words in shadow readme based on Shantanu's advice

#### Change summary
update shadow readme toadd explanation on shadow current capabilities. 

#### Revision diff summary
If there is more than one revision, please explain what has been changed since the last revision.

### Testing
 **Is your change tested? If not, please justify the reason.**  
 **Please list your testing steps and test results.** 
- CI test run result: <link>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by HarshGandhi-AWS — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ The modular IoT Device Client consists of a “base client” and discrete “cl
  Detect feature](https://docs.aws.amazon.com/iot/latest/developerguide/detect-device-side-metrics.html) in the AWS IoT Device Defender service.
 * The client-side [Fleet Provisioning feature](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html) enables you to replace provisional credentials with device-specific ones
   when you onboard a fleet of devices to AWS IoT Core. It creates a device specific certificate and private key, and registers the device on AWS IoT Core.
+* The client-side [Named Shadows feature](https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html) enables you to control your IoT device using AWS IoT Named Shadows. Shadows can store your device's state information and make it available to your device, AWS IoT services, your custom  apps and other AWS services whether the device is online and connected to AWS IoT or not.
```
Comment (satheshantanu) [inline README.md:51]: Please remove the hyperlink from [Named Shadows feature]  and attach it to [AWS IoT Named Shadows]. 
That hyperlink redirects to the cloud side developer guide.

File: README.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ The modular IoT Device Client consists of a “base client” and discrete “cl
  Detect feature](https://docs.aws.amazon.com/iot/latest/developerguide/detect-device-side-metrics.html) in the AWS IoT Device Defender service.
 * The client-side [Fleet Provisioning feature](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html) enables you to replace provisional credentials with device-specific ones
```
Comment (satheshantanu) [inline README.md:49]: Here as well, please remove the hyperlink from [Fleet Provisioning feature]

File: README.md [hunk 3]
```diff
@@ -48,6 +48,7 @@ The modular IoT Device Client consists of a “base client” and discrete “cl
...
   when you onboard a fleet of devices to AWS IoT Core. It creates a device specific certificate and private key, and registers the device on AWS IoT Core.
```
Comment (satheshantanu) [inline README.md:50]: Add the hyperlink here: 
"when you onboard a fleet of devices using the [Fleet Provisioning capability] of AWS IoT Core

**Review by satheshantanu — COMMENTED** [review]
Please address the minor comments in the PR - regarding the readme. (placement of hyperlinks)

**Review by satheshantanu — APPROVED** [review]
Looks great! Ship it!

**Review by HarshGandhi-AWS — APPROVED** [review]

**Pull request merged**